cmd/govim/testdata/scenario_default/quickfix_replaced_dep.txt

```diff
@@ -0,0 +1,63 @@
+# Test that we get diagnostics for edits in a locally replaced dependency
+#
+# The scenario involves having a locally replaced dependency in go.mod.
+# When there is an error introduced into the local copy it should produce
+# a diagnostic (see https://golang.org/issues/36745).
+
+skip 'Pending fix for golang.org/issues/36745'
+
+# Open dependency via GoToDef
+vim ex 'e main.go'
+vim ex 'call cursor(6,11)'
+vim ex 'GOVIMGoToDef'
+errlogmatch '&protocol\.DidOpenTextDocumentParams\{\n\S+:\s+TextDocument: protocol.TextDocumentItem{URI:"file://'$WORK/p/blah.go
```

Comment:
Per #736, instead do something like this:

```
vim expr 'expand(''%:p'')'
stdout '^\Q"'$WORK'/p/blah.go"\E$'
```

You could also check the cursor position as a safety net before making the change.

Reply:
Fixed, don't need to check cursor position here since we just append a specific line.